src/modules/app/containers/I18nProvider.js

```diff
@@ -73,9 +73,9 @@ export class I18nProvider extends React.Component {
 
   loadFonts (language) {
     // Lateef for arabic ui and content, Open Sans for latin text in arabic text, Raleway for latin ui
-    const arabicFonts = ['Lateef:400', 'Raleway:300,400,400i,600,700,700i', 'Open+Sans:400']
+    const arabicFonts = ['Lateef:400', 'Raleway:400,700', 'Open+Sans:400,700']
     // We do not need an arabic font
-    const latinFonts = ['Raleway:300,400,400i,600,700,700i', 'Open+Sans:400']
```

Comment:
Maybe we should load some more Open Sans fonts. The cms can deliver italic and fonts with different font weights so we need to support them.